GenderPayGap.WebJob/Functions/Functions.SendReminderEmails.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autofac;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Core.Classes.Logger;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.Extensions.AspNetCore;
+using Microsoft.Azure.WebJobs;
+using Newtonsoft.Json;
+
+namespace GenderPayGap.WebJob
+{
+    public partial class Functions
+    {
+        public void SendReminderEmails([TimerTrigger("0 * * * *")] TimerInfo timer)
```

Comment:
It might be nice to have a comment to say what this `0 * * * *` means.
I knew you wanted to have it run every hour, but the crontab syntax means I had to scratch my head for a while to check it was right

Reply:
I agree, I've added one in now